src/game.cpp

```diff
@@ -6792,6 +6792,18 @@ Creature const* game::critter_at( const tripoint &p ) const
     return const_cast<game*>(this)->critter_at( p );
 }
 
+monster * game::summon_mon( const std::string id, const tripoint &p )
+{
+    monster mon(GetMType(id));
+    // Set their last upgrade check to the current day.
+    mon.reset_last_load();
+    mon.spawn(p);
+    add_zombie(mon);
+
+    // Return the non-local copy of the monster
+    return &zombie(critter_tracker.mon_at(p));
```

Comment:
Anyone know of a better way to do this than searching through the critter_tracker again to find the monster we just added? I couldn't think of any better way to get a permanent pointer to the monster.